crates/orchestrator/src/tests/server/job_routes.rs

```diff
@@ -0,0 +1,168 @@
+use core::panic;
+use std::net::SocketAddr;
+use std::sync::Arc;
+
+use axum::Router;
+use chrono::{SubsecRound as _, Utc};
+use hyper::{Body, Request};
+use mockall::predicate::eq;
+use rstest::*;
+use starknet::providers::jsonrpc::HttpTransport;
+use starknet::providers::JsonRpcClient;
+use url::Url;
+use utils::env_utils::{get_env_var_or_default, get_env_var_or_panic};
+use uuid::Uuid;
+
+use crate::config::Config;
+use crate::jobs::job_handler_factory::mock_factory;
+use crate::jobs::types::{ExternalId, JobItem, JobStatus, JobType, JobVerificationStatus};
+use crate::jobs::{Job, MockJob};
+use crate::queue::init_consumers;
+use crate::routes::app_routes::{app_router, handler_404};
+use crate::routes::job_routes::job_routes;
+use crate::tests::config::{ConfigType, TestConfigBuilder};
+
+#[fixture]
+async fn setup_job_server() -> (SocketAddr, Arc<Config>) {
```

Comment:
can this be shared with the health route test case?

Reply:
Valid, implemented [here](https://github.com/madara-alliance/madara-orchestrator/pull/164/commits/6cddb0712a31d221286b75c8ee398e0e585fc915).

Reply:
Valid, implemented [here](https://github.com/madara-alliance/madara-orchestrator/pull/164/commits/af460988fe4345796a7ab735017e0fe50ef9ed34).

Thanks @Mohiiit for the structure format : [Here](https://github.com/madara-alliance/madara-orchestrator/pull/164#discussion_r1808360110).